src/app/components/Rofl/AppStatus.tsx

```diff
@@ -0,0 +1,31 @@
+import { ReactNode } from 'react'
+import { useTranslation } from 'react-i18next'
+import CheckCircleIcon from '@mui/icons-material/CheckCircle'
+import { StatusBadge, StatusVariant } from '../common/StatusBadge'
+
+type RoflAppStatusTypes = 'active' | 'inactive' | 'removed'
+
+const statusVariant: Record<RoflAppStatusTypes, StatusVariant> = {
+  active: 'success',
+  inactive: 'warning',
+  removed: 'danger',
+}
+
+const statusIcon: Partial<Record<RoflAppStatusTypes, ReactNode>> = {
+  inactive: <CheckCircleIcon color="warning" fontSize="small" />,
+}
+
+type AppStatusProps = {
+  status: RoflAppStatusTypes
+}
+
+export const AppStatus = ({ status }: AppStatusProps) => {
```

Comment:
Just a nit, but naming for me is a bit misleading, I would prefer it would be `RoflStatusBadge` or something similar.